lib/vtls/wolfssl.c

```diff
@@ -33,8 +33,8 @@
 #ifdef USE_WOLFSSL
 
 #define WOLFSSL_OPTIONS_IGNORE_SYS
-#include <wolfssl/version.h>
 #include <wolfssl/options.h>
+#include <wolfssl/version.h>
```

Comment:
the background on this is like ten years ago some old versions had a buggy options.h and there used to be tests against the version which was only in version.h so it had to be included first

Reply:
Ah, nice. I you think we should leave it as-is, I'm good with it. `version.h` is just two `#define`s:
https://github.com/wolfSSL/wolfssl/blob/39d4832b0b75e4f18521d2841e38f38090adae02/wolfssl/version.h.in#L23-L39

Reply:
it's fine since the checks were removed and i doubt we support versions that old anyway. that's pre-wolfssl time back when it was called cyassl and i doubt we support cyassl anymore.